desktop/apps/armory_week/components/Cta.js

```diff
@@ -42,26 +46,149 @@ const FullWidthCol = styled.div`
   align-items: center;
 `
 
-const Col = styled.div`
+const FixedCol = styled.div`
   display: flex;
   align-items: center;
 `
 
-export const Cta = () =>
-  <StickyFooter>
-    <Container>
-      <Col>
-        <CtaImage src="https://d7hftxdivxxvm.cloudfront.net/?resize_to=fit&width=150&height=149&quality=95&src=https%3A%2F%2Fd32dm0rphc51dk.cloudfront.net%2FE-k-uLoQADM8AjadsSKHrA%2Fsquare.jpg" />
-      </Col>
-      <FullWidthCol style={{ fontSize: "26px" }}>
-        A better way to experience Armory Week 2018
-      </FullWidthCol>
-      <Col>
-        <InvertedButton
-          style={{ marginRight: "30px", width: "160px" }}
-        >
-          Sign Up
-        </InvertedButton>
-      </Col>
-    </Container>
-  </StickyFooter>
+const OverlayModalContainer = styled.div`
+  display: flex;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: white;
+  align-items: center;
+  justify-content: center;
+`
+
+const ContentContainer = styled.div`
+  width: 710px;
+`
+const Close = styled.span`
+  position: absolute;
+  top: 10px;
+  right: 20px;
+  font-size: 40px;
+  color: ${colors.grayMedium};
+  cursor: pointer;
+`
+
+const Separator = styled.div`
+  width: 100%;
+  text-align: center;
+  border-bottom: 1px solid #e5e5e5;
```

Comment:
Minor, but could be: 
`border-bottom: 1px solid ${colors.gray}; ` //or whatever the equivalent was

Reply:
totally missed it I'll send a follow-up PR.

Reply:
sent: https://github.com/artsy/force/pull/2155